node/as/thrift.js

```diff
@@ -65,6 +88,17 @@ function TChannelThriftRequest(options) {
     self.tchannelThrift = options.tchannelThrift;
 }
 
+function health(self, req, head, body, callback) {
+    var status = self.healthCheckCallback();
+    return callback(null, {
```

Comment:
used to provide a health check wrapper for your service, so your service don't have to worry about meta.thrift, etc.